changelogs/unreleased/gh-9917-speed-up-transaction-queue-processing.md

```diff
@@ -0,0 +1,3 @@
+## bugfix/raft
+
+* Optimized transaction queue processing (gh-9917).
```

Comment:
Lets say "synchronous transaction", not just "transaction".

Reply:
Fixed.